packages/client/hmi-client/src/workflow/ops/simulate-ensemble-ciemss/tera-simulate-ensemble-ciemss.vue

```diff
@@ -343,8 +343,10 @@ const runEnsemble = async () => {
 };
 
 onMounted(async () => {
-	// FIXME: probably switch to multiport instead of multivalue
-	const modelConfigurationIds = props.node.inputs[0]?.value;
+	const modelConfigurationIds: string[] = [];
+	props.node.inputs.forEach((ele) => {
```

Comment:
Note not sure how to use `map` cleanly here as at least one ele (the last) will be `undefined` and I dont want to spend time looking into it

Reply:
This is fine.